core/modules/utils/dom/dom.js

```diff
@@ -292,14 +292,17 @@ exports.collectDOMVariables = function(selectedNode,domNode,event) {
 		$tw.utils.each(selectedNode.attributes,function(attribute) {
 			variables["dom-" + attribute.name] = attribute.value.toString();
 		});
-		// Add a variable with a popup coordinate string for the selected node
-		variables["tv-popup-coords"] = "(" + selectedNode.offsetLeft + "," + selectedNode.offsetTop +"," + selectedNode.offsetWidth + "," + selectedNode.offsetHeight + ")";
+		
+		if(selectedNode.offsetLeft && selectedNode.offsetTop && selectedNode.offsetWidth && selectedNode.offsetHeight) {
```

Comment:
Do we need to check for all four? I think the spec probably suggests that if `offsetLeft` exists then you can assume that the others exist.

Reply:
Yes you're right it should be sufficient to check for `offsetLeft` only

Reply:
I've made the check short as it should be